SPECS/rpm/rpm.spec

```diff
@@ -1,21 +1,19 @@
 %{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 %{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%global sover 9
 Summary:        Package manager
 Name:           rpm
-Version:        4.14.2
-Release:        12%{?dist}
+Version:        4.16.1.3
```

Comment:
Can you confirm that `rpm` still builds. I remember [trying to update it](https://github.com/PawelWMS/CBL-Mariner/commit/765b0a07a95ace685009d41fa1a0dfbd8a91e85c) a while ago to version 4.16.0 and I remember encountering many build issues with missing [Perl macros dependencies](https://github.com/PawelWMS/CBL-Mariner/commit/48b2c29cab1541ebaae759995254c37c6e3ae99c), which were no longer provided by `rpm`. I think the newer package also introduced a bunch of new headers and other files.